qiskit_ibm_provider/transpiler/passes/scheduling/__init__.py

```diff
@@ -98,6 +105,272 @@
     dd_teleport.draw(output="mpl")
 
 
+Scheduling old format ``c_if`` conditioned gates
+------------------------------------------------
+
+Scheduling with old format ``c_if`` conditioned gates is not supported.
+
+.. jupyter-execute::
+
+    qc_c_if = QuantumCircuit(1, 1)
+    qc_c_if.x(0).c_if(0, 1)
+    qc_c_if.draw(output="mpl")
+
+The :class:`.IBMBackend` configures a translation plugin
+:class:`.IBMTranslationPlugin` to automatically
+apply transformations and optimizations for IBM hardware backends when invoking
+:class:`qiskit.transpile`. This will automatically convert all old style ``c_if``
```

Comment:
```suggestion
:func:`~qiskit.compiler.transpile`. This will automatically convert all old style ``c_if``
```

Don't ask me why `transpile` is defined in a file called `compiler` (as far as intersphinx is concerned - we promote `qiskit.transpile` to users) but all its passes live in a module called `transpiler` and there's no other mention of the term "compiler" in Terra.  Only madness lies down that path.

Reply:
I know why but its a story for another time 😉 . Fixed in f8598cab3897463025062bc72dfa2711d996eb3d.